pkgs/development/compilers/gcc/12/default.nix

```diff
@@ -288,7 +288,10 @@ lib.pipe (stdenv.mkDerivation ({
 
   configurePlatforms = [ "build" "host" "target" ];
 
-  configureFlags = callFile ../common/configure-flags.nix { };
+  configureFlags = callFile ../common/configure-flags.nix { }
+    # On mips64-unknown-linux-gnu libsanitizer defines collide with
+    # glibc's definitions and fail the build. It was fixed in gcc-13+.
+    ++ lib.optionals targetPlatform.isMips [ "--disable-libsanitizer" ];
```

Comment:
You'd rather disable it across the board on MIPS, rather than just the MIPS platforms it affects (IIRC, ones without an explicit ABI)?  (I have no opinion — you're the one who actually uses MIPS!)

Reply:
Good point, thanks for noticing!

Fixed in dca71d50d29d